File: movies-api/src/main/java/com/ivanfranchin/moviesapi/exception/UserExtraNotFoundException.java
package com.ivanfranchin.moviesapi.exception;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@ResponseStatus(HttpStatus.NOT_FOUND)
public class UserExtraNotFoundException extends RuntimeException {

    public UserExtraNotFoundException(String username) {
        super("UserExtra of %s not found".formatted(username));
    }
}

File: movies-api/src/main/kotlin/com/ivanfranchin/moviesapi/userextra/exception/UserExtraNotFoundException.kt
package com.ivanfranchin.moviesapi.userextra.exception

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

@ResponseStatus(HttpStatus.NOT_FOUND)
class UserExtraNotFoundException(username: String?) :
    RuntimeException("UserExtra of $username not found")
